### PR TITLE
add LocationBFDR and used it

### PR DESCRIPTION
**Pull request by saulakravitz**
Integration with Data Dictionary remains to be done.

### CODE REVIEW

**Comment by rpassas** [comment]
Made a small edit to a typo that prevented building.

I would update the changelog too.

This branch has some errors/warnings that seem unrelated to changes being made (at least in local build). Some might need an ignorewarning update or tweaking of examples. Others might just be an issue with the terminology server.

Examples:

**UCUM warning**
WARNING: Bundle/bundle-jurisdiction-fetal-death-not-named: Bundle.entry[11].resource.value.ofType(Quantity).code: Resolved system http://unitsofmeasure.org, but the definition is not complete, so assuming value set include is correct

**TX server Issue**
ERROR: Bundle/bundle-provider-fetal-death-not-named: Bundle.entry[33].resource/*QuestionnaireResponse/QuestionnaireResponse-patients-fetal-death-carmen-lee*/.item[7].item[0].answer[0]: The code provided 2135-2 in the system urn:oid:2.16.840.1.113883.6.238) is not in the options value set (OMB Ethnicity Categories) in the questionnaire: Error reading Http Response from http://tx.fhir.org/r4: Error parsing JSON source: Unexpected char '<' in json stream at Line 1 (path=[null])

**Also TX server Issue (?)**
WARNING: Bundle/bundle-provider-live-birth-babyg-quinn: Bundle.entry[54].resource/*QuestionnaireResponse/QuestionnaireResponse-mothers-live-birth-jada-quinn*/.item[8].item[0].answer[0]: The code provided 2186-5 cannot be validated in the options value set (OMB Ethnicity Categories) in the questionnaire because the system urn:oid:2.16.840.1.113883.6.238 is unknown

File: input/fsh/instances/location-north-hospital.fsh [hunk 1]
```diff
@@ -1,8 +1,9 @@
 Instance: location-north-hospital
-InstanceOf: USCoreLocation
+InstanceOf: LocationBFDR
 Title: "US Core Location - North Hospital"
 Description: "US Core Location: North Hospital example"
```
Comment (rpassas) [inline input/fsh/instances/location-north-hospital.fsh]: and here

File: input/fsh/instances/location-south-hospital.fsh [hunk 2]
```diff
@@ -1,5 +1,5 @@
 Instance: location-south-hospital
-InstanceOf: USCoreLocation
+InstanceOf: LocationBFDR
 Title: "US Core Location - South Hospital"
 Description: "US Core Location: South Hospital example"
 Usage: #example
```
Comment (rpassas) [inline input/fsh/instances/location-south-hospital.fsh:5]: same here

File: input/fsh/instances/location-east-hospital.fsh [hunk 3]
```diff
@@ -1,8 +1,9 @@
 Instance: location-east-hospital
-InstanceOf: USCoreLocation
+InstanceOf: LocationBFDR
 Title: "US Core Location - East Hospital"
 Description: "US Core Location: East Hospital example"
 Usage: #example
```
Comment (rpassas) [inline input/fsh/instances/location-east-hospital.fsh:5]: I would update the title/description to not be "US Core Location"

**Comment by saulakravitz** [comment]
The terminology issues look bogus/transient to me.
There were other real issues....

File: input/fsh/profiles/EncounterBirth.fsh [hunk 4]
```diff
@@ -32,7 +32,7 @@ Description: "This Encounter profile represents the child's birth encounter."
   * individual 1.. MS
   * individual only Reference(PractitionerVitalRecords)
 * hospitalization MS
-  * destination only Reference(USCoreLocation)
+  * destination only Reference(LocationBFDR)
   * destination MS
     * ^short = "If infant transferred, facility transferred to"
     * ^definition = "If infant transferred, facility transferred to"
```
Comment (rpassas) [inline input/fsh/profiles/EncounterBirth.fsh:38]: "HOSPTO" is mapped to hospitalization.destination.name - we should have a short description in encounter birth so it shows in the differential

File: input/fsh/profiles/EncounterBirth.fsh [hunk 5]
```diff
@@ -42,7 +42,7 @@ Description: "This Encounter profile represents the child's birth encounter."
 * location MS
   * ^short = "Information about the location of birth"
   * ^definition = "Information about the location of birth"
-  * location only Reference(USCoreLocation)
+  * location only Reference(LocationBFDR)
   * location MS
     * ^short = "Facility name (if not institution, give street and number)"
     * ^definition = "Facility name (if not institution, give street and number)"
```
Comment (rpassas) [inline input/fsh/profiles/EncounterBirth.fsh]: "HOSP" is mapped to location.location.name - we should have a short description in encounter birth so it shows in the differential

File: input/fsh/profiles/Encounter_Maternity.fsh [hunk 6]
```diff
@@ -46,7 +46,7 @@ Description: "This Encounter profile represents the mother's maternity encounter
 * location MS
   * ^short = "Information about the location of delivery"
   * ^definition = "Information about the location of devivery"
-  * location only Reference(USCoreLocation)
+  * location only Reference(LocationBFDR)
   * location MS
     * ^short = "Facility name (if not institution, give street and number)"
     * ^definition = "Facility name (if not institution, give street and number)"
```
Comment (rpassas) [inline input/fsh/profiles/Encounter_Maternity.fsh:52]: "HOSP_D" maps to location.location.name - not currently in differential

Comment (rpassas) [reply input/fsh/profiles/Encounter_Maternity.fsh:52]: Several other fields should have comments to show in the differential:
-	Encounter maternity: location.location.address.extension[XXXX] + maps to various address-related codes 
-	Encounter maternity: location.location.address.line/postalCode/district/city map to various location codes
-	Encounter maternity: “CNTYO” mapped to location.location.address.district.extension[countyCode] 
-	Encounter maternity: “HOSPFROM” mapped to hospitalization.origin.name

**Comment by rpassas** [comment]
Currently, the location-related content (Birth and Fetal Death Location, ValueSet - Location Types, Location Type Code System) appears in the misc. piles at the bottom of the artifacts page; we should put these in the appropriate sections in the sushi-config

File: input/fsh/profiles/Encounter_Maternity.fsh (hunk 6 above)
Comment (saulakravitz) [reply input/fsh/profiles/Encounter_Maternity.fsh:52]: These map to LocationBFDR

File: input/fsh/profiles/EncounterBirth.fsh (hunk 4 above)
Comment (saulakravitz) [reply input/fsh/profiles/EncounterBirth.fsh:38]: Maps to LocationBFDR

**Review by rpassas — APPROVED** [review]

**Pull request merged**